README.md

```diff
@@ -14,9 +14,14 @@ formatted text, math, and multimedia in a single document
 
 First, [install IPython](http://ipython.org/install.html); you may
 also want some scientific-Python packages (SciPy and Matplotlib).
-The simplest way to do this on Mac and Windows is by [downloading the
-Anaconda package](http://continuum.io/downloads) and running its
-installer.  (Do *not* use Enthought Canopy/EPD.)
+The simplest way to do this on Mac and Windows is by using the
+Conda.jl package that installs the [Miniconda](http://conda.pydata.org/miniconda.html) Python distribution. Alternativly [Anaconda](http://continuum.io/downloads) can be installed. (Do *not* use Enthought Canopy/EPD.) To install Miniconda do:
+```
+Pkg.add("Conda")
+using Conda
+# Installing the notebook interface
+Conda.add("ipython-notebook") # important restart the julia terminal before running Pkg.add("IJulia")
```

Comment:
Yes, probably:

* The Conda package should export something like `Conda.binpath("jupyter")` to return the absolute path of a conda-installed executable.
* `IJulia/deps/jupyter.jl` should look in `JUPYTER` environment variable or similar(?) to allow an absolute path to be user-specified at `Pkg.build`-time (without modifying `PATH`)
* The Juptyer path should be output to a `const jupyter = "..."` in a `IJulia/deps/deps.jl` file that is generated during `Pkg.build`, and is `included` from `IJulia.jl`.